sed/config/default.yaml

```diff
@@ -95,6 +95,10 @@ delay:
   # value ranges of the analog-to-digital converter axes used for encoding the delay stage position
   # (in unbinned coordinates)
   adc_range: [1900, 25600]
+  # pump probe time overlap in ps
+  time0: 0
+  # if to flip the time axis
+  flip_time_axis: False
 
```

Comment:
add defaults for newly introduced dataframe columns, if you want to keep them

Reply:
In the default config

Reply:
this is `sed/config/default.yaml`. isn't this the default config?

Reply:
Yes. Sorry, misinterpreted my own comment. What I mean is that you should add values for newly added dataframe columns to this file. E.g. delayStageAxis or how you called it. But if there is a mechanism that you don't need it, fine.

Reply:
I think most of the features I implemented should correctly catch errors when relevant columns are missing. I prefer this in general, over a default setting which might be different from what expected.
I'd keep defaults for things like number of cores, histogramming options etc...